src/EncompassRest/Loans/LoanOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using EncompassRest.Utilities;
+
+namespace EncompassRest.Loans
+{
+    public abstract class LoanOptions
+    {
+        public bool Populate { get; set; }
+
+        public string LoanTemplate { get; set; }
+
+        internal LoanOptions()
+        {
+        }
+
+        internal virtual QueryParameters ToQueryParameters()
+        {
+            var queryParameters = new QueryParameters();
+            if (Populate)
+            {
+                queryParameters.Add("view", "entity");
```

Comment:
queryParameters.Add("view", populate?"entity":"id");

Reply:
We don't need "id" as we'll get the id from the location header.

Reply:
the 2 options for view are entity or id.  should we pass it always or only when not default?

Reply:
id is no use to us as it's included in the header thus I think we should only support entity when using populate and don't include a view query parameter at all if not populate.

Reply:
ok